features/ALS006_Alignment-horizontal-shape-representation.feature

```diff
@@ -0,0 +1,12 @@
+@implementer-agreement
+@ALS
+Feature: ALS006 - Horizontal alignment shape representation
+  Scenario: Agreement on each IfcAlignmentHorizontal using correct representation
+
+      Given A file with Schema Identifier "IFC4X3_TC1" or "IFC4X3_ADD1" or "IFC4X3"
+      And An IfcAlignmentHorizontal
+      And Its attribute Representation
+      And Its attribute Representations
+      Then The value of attribute RepresentationIdentifier must be Axis
+      And  The value of attribute RepresentationType must be Curve2D
+      And  The type of attribute Items must be IfcCompositeCurve or IfcIndexedPolycurve or IfcPolyline
```

Comment:
Comparing lines 6 and 12: how come one needs `"`, while the other doesn't? This seem inconsistent and imho unnecessarily complicates the rule writing / reading.

Reply:
I understand how it might appear inconsistent. We're currently experimenting to find a balance between ease of Python implementation and the human readability of Gherkin statements.

Currently, it's due to the following given statement
https://github.com/buildingSMART/ifc-gherkin-rules/blob/05329ef21559fc1de7113626f716ead32a5b0320/features/steps/givens/attributes.py#L34

This implementation results in 'Schema Identifier' being parsed into the 'field' variable, and the IFC versions into the 'values' variable. Additionally, the 'field' variable is used to parse the value 'Model View Definition' within the same function. The " symbol is essential for Gherkin to delineate where the 'field' variable concludes and 'values' begins.

For instance, with 
https://github.com/buildingSMART/ifc-gherkin-rules/blob/05329ef21559fc1de7113626f716ead32a5b0320/features/SPS001_Basic-spatial-structure-for-buildings.feature#L15-L16

I'll merge this PR and move the discussion to the 'custom parsing' issue and PR. I hope that's ok. 
https://github.com/buildingSMART/ifc-gherkin-rules/pull/48
https://github.com/buildingSMART/ifc-gherkin-rules/issues